plotly/tools.py

```diff
@@ -1570,6 +1578,43 @@ def _validate_streamline(x, y):
                                              "evenly spaced array")
 
     @staticmethod
+    def _validate_annotated_heatmap(z, annotation_text):
+        """
+        Annotated heatmap specific validations
```

Comment:
Small :goat:, but I think you need hyphens here. `Annotated-heatmap-specific validations`. Or, `Validations specific to annotated heatmaps`?